src/core/client/stream/tabs/Comments/Comment/EditCommentForm/EditCommentForm.tsx

```diff
@@ -70,6 +70,7 @@ const EditCommentForm: FunctionComponent<EditCommentFormProps> = props => {
               name="body"
               validate={getCommentBodyValidators(props.min, props.max)}
             >
+              {/* FIXME: (wyattjoh) reorganize this */}
```

Comment:
Was this left in as a personal reminder, or as a "defer util later" notice?

Reply:
`CORL-492`